src/praisonai-agents/tests/unit/test_stream_sampling_kwargs.py

```
@@ -0,0 +1,90 @@
+
+from praisonaiagents import Agent
+
+
+class _Delta:
+    content = None
+    tool_calls = None
+
+
+class _Chunk:
+    def __init__(self):
+        d = _Delta(); d.content = "ok"
```

Ruff (0.16.3)

[error] Multiple statements on one line (semicolon)

(E702)